GOVERNANCE.md

```diff
@@ -107,6 +107,28 @@ Lazy consensus does _not_ apply to the process of:
 
 * Removal of maintainers from Velero
 
+## Deprecation Policy
+
+### Deprecation Process
+
+Any contributor may introduce a request to deprecate a feature or an option of a feature by opening a feature request issue in the vmware-tanzu/velero GitHub project. The issue should describe why the feature is no longer needed or has become detrimental to velero, as well as whether and how it has been superseded. The submitter should give as much detail as possible.
+
+Once the issue is filed, a discussion period begins which ends at the beginning of the second community meeting after the opening of the issue, and is held at the community meeting and on the issue itself; the person who opens the issue or a maintainer should add that date and time in a comment in the issue as soon after the issue is opened as possible.
```

Comment:
I don't think this discussion period of two community meetings should be a hard-and-fast rule, as the deprecation/removal of some features may be more involved than others. In particular, there may be cases where we need to think through the upgrade path for the community.

Reply:
We have to introduce some hard lines otherwise it can take forever to be reviewed. Two community  meetings are 2 weeks so maybe that's a bit short indeed, maybe we can word it as "Two consecutive meetings in the same timezone" which will make it a month and that's in total 4 meetings

Reply:
@OrlinVasilev It might be simpler to just set the time to "one month" -- in practice, that means 2 meetings per timezone, but it's easier to understand a month vs. "2 meetings in one time zone".

Reply:
Agree as I said month is easier also we can track it with github time stamps

Reply:
@weshayutin will you be able to make this change?

Reply:
Done, updated